README.md

```diff
@@ -22,7 +28,120 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+### `FastStats::Means`
+
+Collect means for multiple metrics:
+
+### Initialize
+
+```ruby
+means = FastStats::Means.new
+```
+
+#### Instance Methods
+
+**`add(metric_name, val) ->  value`**<br>
+    Adds the `val` to the `name` mean and returns the new `n` (count) for that mean. <br>
```

Comment:
If you use prefix the function description with `-` here it should do the indentation you want.  Using 4 spaces doesn't appear to be working here.

Reply:
Or use another level of nesting for the markdown headers, e.g.
```
#### Instance Methods

##### add(metric_name, val) ->  value
```

Reply:
@andreafrost I don't know if I follow your first comment here. I am avoiding the `####` because that adds headers spacing below and some of these have multiple signatures I wanted listed together. I also specifically did not use bullets to be as consistent with the Ruby Doc as possible.

Reply:
Oh, I see this was accidentally indented. There should be no indentation here.